src/@next/Carousel/Carousel.tsx

```diff
@@ -30,8 +30,10 @@ export interface CarouselProps {
   width?: string;
   indicatorPosition?: 'top' | 'bottom' | 'left' | 'right' | 'outer';
   indicatorType?: 'dot' | 'line' | 'slider';
-  /** When to show arrows*/
-  showArrow?: 'always' | 'hover' | 'never';
+  /** Sets whether to show indicator */
+  showIndicator?: boolean;
```

Comment:
can we use `indicatorType` for this instead? If it's falsy we don't show it

Reply:
yep, good idea

Reply:
I think it's okay to have default value just assign it to null if not needed and the prop is optional anyway right?